fiftyone/utils/sapiens.py

```
@@ -0,0 +1,296 @@
+        "sapiens2_1b_pose.safetensors",
+    ),
+    "facebook/sapiens2-pose-5b": (
+        "sapiens2_5b",
+        "sapiens2_5b_pose.safetensors",
+    ),
+}
+
+_SAPIENS_REQ = "sapiens @ git+https://github.com/facebookresearch/sapiens2.git"
+
+
+def _ensure_sapiens():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_ensure_sapiens`

Add return type annotation: `None`

(ANN202)